### PR TITLE
Grammar update to capturing.rst

### PR DESCRIPTION
**Pull request by ohyeah2389**
I fixed the grammar and improved the readability of the document.

### CODE REVIEW

**Review by natowi — APPROVED** [review]

File: source/capturing/capturing.rst [hunk 1]
```diff
@@ -3,24 +3,24 @@ Capturing
...
-- Avoid moving objects in the scene or background!
-- Rotate only objects with a plain background
-- The Object of interest should always fill most of the image
-- Take images with a side overlap of min. 60% and frontal overlap of 80%.
-- For each shot, move to a new position (or rotate the object)
+- Your scene/object should be well lit. 
+- Avoid shadows, reflections, and transparent objects.
+- Take the photos in diffuse or indirect lighting, such as on an overcast day (if outside) or using multiple light sources (if inside).
+- Don´t use the flash setting on the camera.
+- Do not change the focal length/zoom while shooting. Use a fixed lens if possible.
```
Comment (simogasp) [inline source/capturing/capturing.rst]:
```suggestion
- Do not change the focal length/zoom while shooting. Use a fixed focal length lens if possible.
```

File: source/capturing/capturing.rst [hunk 2]
```diff
@@ -3,24 +3,24 @@ Capturing
...
-- Do not change the focal length while shooting or use a fixed lens.
-- Make sure you can take pictures from all angles.
-- Avoid moving objects in the scene or background!
-- Rotate only objects with a plain background
-- The Object of interest should always fill most of the image
-- Take images with a side overlap of min. 60% and frontal overlap of 80%.
-- For each shot, move to a new position (or rotate the object)
+- Your scene/object should be well lit. 
+- Avoid shadows, reflections, and transparent objects.
+- Take the photos in diffuse or indirect lighting, such as on an overcast day (if outside) or using multiple light sources (if inside).
```
Comment (simogasp) [inline source/capturing/capturing.rst]:
```suggestion
- Take the photos in diffuse or indirect lighting, such as on an overcast day (outdoor) or using multiple light sources (indoor).
```

File: source/capturing/capturing.rst [hunk 3]
```diff
@@ -3,24 +3,24 @@ Capturing
...
+- The object of interest should always fill most of the image.
+- Take images with a side overlap of 60% minimum and frontal overlap of 80% minimum.
+- For each shot, move to a new position (or rotate the object).
 - Do not take multiple images from the same spot.
-- You can photograph multiple times in different patterns to leave no blind spots
-- avoid shaking
-- The more images you have, the better. You can always decide not to use them…
+- You can photograph an area multiple times in different patterns for better coverage.
+- Avoid shaky, blurry, or warped images.
+- The more images you have, the better. You can always filter out repetitive or bad images to reduce processing time.
```
Comment (simogasp) [inline source/capturing/capturing.rst]:
```suggestion
- The more images you have, the better. You can always filter out repetitive or poor quality images to reduce processing time.
```

File: source/capturing/capturing.rst [hunk 4]
```diff
@@ -3,24 +3,24 @@ Capturing
...
+- Avoid moving objects in the scene or background.
+- If taking pictures using a rotating rig, make sure to use a plain color background with no distiguishable features.
+- The object of interest should always fill most of the image.
+- Take images with a side overlap of 60% minimum and frontal overlap of 80% minimum.
+- For each shot, move to a new position (or rotate the object).
 - Do not take multiple images from the same spot.
-- You can photograph multiple times in different patterns to leave no blind spots
-- avoid shaking
-- The more images you have, the better. You can always decide not to use them…
+- You can photograph an area multiple times in different patterns for better coverage.
```
Comment (simogasp) [inline source/capturing/capturing.rst]:
```suggestion
- For better coverage, you can photograph an area multiple times in different acquisition patterns.
```

**Review by simogasp — CHANGES_REQUESTED** [review]
Thanks for this!
I suggested some other minor changes

**Pull request merged**